client/app/account/billing/mean/add/billing-mean-add.html

```diff
@@ -96,57 +96,147 @@
-                        <label for="ownerName"
-                               class="control-label required"
-                               data-translate="paymentMean_label_ownerName">
-                        </label>
-                        <input type="text"
-                               id="ownerName"
-                               name="ownerName"
-                               class="form-control"
-                               data-ng-model="mean.ownerName"
-                               data-ng-minlength="5"
-                               data-ng-maxlength="255"
-                               required/>
-                        <div data-ng-repeat="(error, hasError) in paymentMeanForm.ownerName.$error"
-                             role="alert">
-                            <span class="text-danger"
-                                  data-ng-bind="('paymentMean_error_' + error) | translate"
-                                  data-ng-if="hasError && paymentMeanForm.ownerName.$dirty">
-                            </span>
+                        <div
+                            data-ng-if="!isFrenchCustomer"
+                            class="form-group">
+                            <label for="ownerAddress"
+                                    class="control-label required"
+                                    data-translate="paymentMean_label_ownerAddress">
+                            </label>
+                            <input type="text"
+                                    id="ownerAddress"
+                                    name="ownerAddress"
+                                    class="form-control"
+                                    data-ng-model="mean.ownerAddress"
+                                    data-ng-maxlength="255"
+                                    required>
+                            <div data-ng-repeat="(error, hasError) in paymentMeanForm.ownerAddress.$error"
+                                    role="alert">
+                                <span class="text-danger"
+                                        data-ng-bind="('paymentMean_error_' + error) | translate"
+                                        data-ng-if="hasError && paymentMeanForm.ownerAddress.$dirty" >
+                                </span>
+                            </div>
                         </div>
-                    </div>
 
+                        <div class="form-group">
+                            <label for="ownerName"
+                                class="control-label required"
+                                data-translate="paymentMean_label_ownerName">
+                            </label>
+                            <input type="text"
+                                id="ownerName"
+                                name="ownerName"
+                                class="form-control"
+                                data-ng-model="mean.ownerName"
+                                data-ng-minlength="5"
```

Comment:
my name is "jet li" ok it does work ...

Reply:
I do agree with you, even if your company is named "OVH" you can't add a means of payment, but these are the rules currently in place and I didn't want to change them without confirmation first